spec/unit/messages/users_list_message_spec.rb

```diff
@@ -84,10 +99,32 @@ module VCAP::CloudController
         expect(message).to be_valid
       end
 
+      it 'accepts partial_usernames and origins' do
```

Comment:
Also rename the parent context/describe to `origin_requires_username_or_partial_username`?